src/storage/src/sink/kafka.rs

```diff
@@ -1091,195 +1084,184 @@ where
         }
         *s.healthchecker.lock().await = healthchecker;
 
-        tokio::pin!(shutdown_rx);
-        loop {
-            tokio::select! {
-                _ = shutdown_rx.as_mut() => {
-                    debug!("shutting down sink: {}", &s.name);
-
-                    // Approximately one last attempt to push anything pending to kafka before closing.
-                    if !shutdown_flush.get() {
-                        debug!("flushing kafka producer for sink: {}", &s.name);
-                        let _ = s.producer.flush().await;
-                        shutdown_flush.set(true);
-                    }
```

Comment:
this was just best-effort right? so we are just removing it?

Reply:
exactly